lib/cylc/cfgspec/glbl_cfg.py

```diff
@@ -18,7 +18,7 @@
 """Allow lazy loading of `cylc.cfgspec.globalcfg`."""
 
 
-def glbl_cfg():
+def glbl_cfg(cached=True):
```

Comment:
Possibly yes, the reason is was done this way is so that the global config is loaded lazily (on demmand). Whether there is any advantage whatsoever in this case is, however, a different matter.

The change was made in #2601.